### DISCUSSION TITLE
zh/faq/

### DISCUSSION THREAD

**Discussion by giscus** [mineadmin/MineAdmin#365] [Q&A]
# zh/faq/

企业级后台管理系统

https://doc.mineadmin.com/zh/faq/

**Comment by Mai-king** [comment]
希望组件能视图化

**Reply by kanyxmo** [reply]
没太明白

**Comment by andphp** [comment]
新版没有CURD生成工具吗，

**Reply by kanyxmo** [reply]
有的，这几天代码生成器就会上架应用市场以插件形式发布（免费的）

**Comment by guanzhao** [comment]
以前得数字字典用起来非常好,新版还会有吗？

**Reply by zds-s** [reply]
会，字典插件

**Reply by kanyxmo** [reply]
数据字典插件已经上架了

**Comment by cesarcool** [comment]
我想用mineadmin，替换我现在的后台系统。唯一的麻烦是登录的用户表，这个可以做到登录用户表用自己老系统的吗。

**Reply by zds-s** [reply]
支持

**Comment by misakimisakimei** [comment]
字典插件什么时候发布?

**Reply by kanyxmo** [reply]
发布了

**Comment by hendulangzi** [comment]
已经配置了token了，还是提示这个 目前无法获取到应用商店数据，请先设置 MINE_ACCESS_TOKEN。

**Reply by zds-s** [reply]
> 已经配置了token了，还是提示这个 目前无法获取到应用商店数据，请先设置 MINE_ACCESS_TOKEN。

可以添加一下qq群，在群里反馈一下完整的操作步骤和上下文

**Comment by hendulangzi** [comment]
Plugin_MineAdmin_AppStore_Controller_IndexController.proxy.php): Failed to open stream: No such file or directory
提示这个，怎么处理

**Reply by zds-s** [reply]
重启服务端
